app/tests/e2e/test_revocation.py

```diff
@@ -17,9 +17,10 @@
 
 @pytest.mark.anyio
 async def test_clear_pending_revokes(
-    faber_client: RichAsyncClient, issue_revocable_credentials_to_alice_and_revoke: list
+    faber_client: RichAsyncClient,
+    issue_revocable_credentials_to_alice_and_revoke_unpublished: list,
```

Comment:
In this case it would be `List[dict]` but yes I agree. I will fix this in a new PR, I don't what to block you.

Reply:
No prob. It's always good to have descriptive types. If it's a `dict`, it's better described: `Dict[str,str]`, otherwise the compiler thinks it's working with `Any`. I'll patch it